spec/rubocop/cop/style/raise_args_spec.rb

```diff
@@ -237,6 +237,60 @@ def do_something
       end
     end
 
+    context 'when exception type is in AllowedNonExplodedTypes' do
+      before do
+        stub_const('Ex1', StandardError)
+        stub_const('Ex2', StandardError)
+      end
+
+      let(:cop_config) do
+        {
+          'EnforcedStyle' => 'exploded',
+          'AllowedNonExplodedTypes' => [%w[Ex1 Ex2], ['Ex1']].sample
```

Comment:
I don't think we have any other specs where some of the set-up is random. We usually do shared examples and test both (all) cases every time.

Reply:
My intention was to show that all of the options are equivalent, without wasting the processing time. But, I understand that this isn't common so I'm happy to switch to 2 different tests.